src/dashbls/depends/relic/include/relic_util.h

```diff
@@ -253,7 +253,7 @@
 /*============================================================================*/
 
 /**
- * Toggle endianess of a digit.
+ * Toggle endianness of a digit.
```

Comment:
same reason but this time the origin is relic lib

Reply:
Thank you and sorry for not noticing that. I am going to fix it upstream.